docs/source/use-cases.rst

```diff
@@ -248,38 +248,38 @@ API requests used:
 
 .. _deposit_funds_to_ex:
 
-Use case: Deposit funds to exchange
+Use case: Deposit funds to an exchange
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Let's not make sphinx too nervous.

/Users/syem/ins-docs/docs/source/use-cases.rst:252: WARNING: Title underline too short.

Use case: Deposit funds to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:252: WARNING: Title underline too short.

Use case: Deposit funds to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:264: WARNING: Title underline too short.

Use case: Deposit coins to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:264: WARNING: Title underline too short.

Use case: Deposit coins to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:276: WARNING: Title underline too short.

Use case: Withdraw coins from an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:276: WARNING: Title underline too short.

Use case: Withdraw coins from an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reply:
fixed all the cases in use-cases.rst